src/widgets/DateRangeSelector.js

```diff
@@ -4,81 +4,88 @@
  * Sustainable Solutions (NZ) Ltd. 2020
  */
 import React from 'react';
-import { Text, StyleSheet } from 'react-native';
+import { Text, StyleSheet, View } from 'react-native';
 import PropTypes from 'prop-types';
 import moment from 'moment';
 
 import { DatePickerButton } from './DatePickerButton';
 import { FlexRow } from './FlexRow';
-import { FlexColumn } from './FlexColumn';
 
-import { textStyles, SUSSOL_ORANGE } from '../globalStyles';
-import { generalStrings } from '../localization/index';
+import { textStyles, LIGHT_GREY, WHITE } from '../globalStyles';
+import { useLayoutDimensions } from '../hooks/useLayoutDimensions';
 
 export const DateRangeSelector = ({
   initialStartDate,
   initialEndDate,
   onChangeFromDate,
   onChangeToDate,
-  labelTextStyle,
   dateTextStyle,
   maximumDate,
   minimumDate,
+  backgroundColor,
 }) => {
   const formattedStartDate = moment(initialStartDate).format('D/M/YYYY');
   const formattedEndDate = moment(initialEndDate).format('D/M/YYYY');
+  const [, height, setDimensions] = useLayoutDimensions();
 
   return (
-    <FlexRow flex={1} alignItems="center">
+    <View
+      onLayout={setDimensions}
+      style={[localStyles.container, { borderRadius: height, backgroundColor }]}
```

Comment:
Does a borderRadius equal to the height make a circle or something? I have 0 understanding how border radius' (radii??) work and just put in numbers till look good, usually

Reply:
I had to look... [CSS borderRadius is wild](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius). But generally the rule of thumb for a circle is half height or width is the correct radius. But if you over shoot it doesn't make a difference. So I didn't bother halving. Possibly bad haha.

How I figure it works is probably easiest drawn... I'm going to show you IRL cause my drawing is EMBARRASSING.